12_listcomprehension_lab/01.py

```diff
@@ -4,5 +4,7 @@
 Hint: look for chr() and ord()
 """
 
+lowletters = [chr(l) for l in range(255) if chr(l).islower()]
 
+print lowletters
 
```

Comment:
Maybe that islower protects you here, but it would be much better to have the range run only on the characters you'd like to print and not all the ascii characters.

BTW Have you tried printing the characters under 32 ascii? It is spectacular.